frontend/dispatcher.lua

```diff
@@ -551,17 +562,14 @@ end
 function Dispatcher:menuTextFunc(settings)
     local action_name = _("Pass through")
     if settings then
-        local count = util.tableSize(settings)
+        local count = Dispatcher:_itemsCount(settings)
         if count == 0 then return _("Nothing") end
-        if count > 1 and settings.settings ~= nil then
-            count = count - 1
-        end
         if count == 1 then
             local item = next(settings)
             if item == "settings" then item = next(settings, item) end
             action_name = Dispatcher:getNameFromItem(item, settings)
         else
-            action_name = _("Many")
+            action_name = T(_("%1 actions"), count)
```

Comment:
http://koreader.rocks/doc/modules/gettext.html#ngettext

Reply:
What if this is already in a > 1 branch ? Put something anyway for the singular form for translators to translate even if not used ? Put an empty string ?
And how do we now how many plural form to put here, as its a property of the language. Or do we just use it with 2 items (as the doc says,  `ngettext (msgid, msgid_plural, n) `), and it's the translation tools that will propose and record the 5 plural variations of Tadjugizbekistanese ?

Reply:
> What if this is already in a > 1 branch ?

That's my entire point. In some languages you have 3many vs 5many vs 10many vs 100many.

Reply:
> And how do we now how many plural form to put here, as its a property of the language. Or do we just use it with 2 items (as the doc says, ngettext (msgid, msgid_plural, n) ), and it's the translation tools that will propose and record the 5 plural variations of Tadjugizbekistanese ?

Yes, that's what it's for. English (as well as Dutch, French, etc.) basically only have 1 or plural.

Reply:
> Put something anyway for the singular form for translators to translate even if not used ? Put an empty string ?

Probably empty.